README.md

```diff
@@ -7,26 +7,13 @@ Ruby LSP Rails is a [Ruby LSP](https://github.com/Shopify/ruby-lsp) addon for ex
 
 ## Installation
 
-To install, add the following line to your application's Gemfile:
-
-```ruby
-# Gemfile
-group :development do
-  gem "ruby-lsp-rails"
-end
-```
+As of v0.3.0, [Ruby LSP](https://github.com/Shopify/ruby-lsp) will automatically include the Ruby LSP Rails addon in its custom bundle when a Rails app is detected. There is no need to add the gem to your bundle.
```

Comment:
I think for newcomers it'd be beneficial to explicitly say to use Ruby LSP Rails you need to follow https://github.com/shopify/ruby-lsp?tab=readme-ov-file#usage to setup `Ruby LSP` with your editor.

Reply:
Good point, added.